### PR TITLE
config fix for ui ldap auth source tests

### PR DESCRIPTION
**Pull request by sthirugn**
```sh
----------------------------------------------------------------------
Ran 14 tests in 191.462s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_ldapauthsource.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def setUpClass(cls):
         super(LDAPAuthSource, cls).setUpClass()
         # TODO: handle when the ldap config is not available
         cls.ldap_user_name = settings.ldap.username
-        cls.ldap_user_passwd = settings.ldap.passwd
```
Comment (elyezer) [inline tests/foreman/ui/test_ldapauthsource.py]: :+1:

**Comment by JacobCallahan** [comment]
ACK

**Comment by elyezer** [comment]
ACK

**Comment by sthirugn** [comment]
part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**